libs/ui/src/lib/symbol-autocomplete/symbol-autocomplete.component.ts

```diff
@@ -0,0 +1,150 @@
+import { FocusMonitor } from '@angular/cdk/a11y';
+import {
+  ChangeDetectionStrategy,
+  ChangeDetectorRef,
+  Component,
+  ElementRef,
+  Input,
+  OnDestroy,
+  OnInit,
+  ViewChild
+} from '@angular/core';
+import { FormControl, NgControl, Validators } from '@angular/forms';
+import { MatAutocompleteSelectedEvent } from '@angular/material/autocomplete';
+import { MatFormFieldControl } from '@angular/material/form-field';
+import { MatInput } from '@angular/material/input';
+import { LookupItem } from '@ghostfolio/api/app/symbol/interfaces/lookup-item.interface';
+import { isString } from 'lodash';
+import { Observable, Subject, of } from 'rxjs';
+import {
+  debounceTime,
+  distinctUntilChanged,
+  switchMap,
+  takeUntil
+} from 'rxjs/operators';
+
+import { DataService } from '../../../../../apps/client/src/app/services/data.service';
+import { AbstractMatFormField } from '../abstract-mat-form-field';
+
+@Component({
+  host: {
+    '[id]': 'id',
+    '[attr.aria-describedBy]': 'describedBy'
+  },
+  selector: 'gf-symbol-autocomplete',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  styleUrls: ['./symbol-autocomplete.component.scss'],
+  templateUrl: 'symbol-autocomplete.component.html',
+  providers: [
+    {
+      provide: MatFormFieldControl,
+      useExisting: SymbolAutocompleteComponent
+    }
+  ]
+})
+export class SymbolAutocompleteComponent
+  extends AbstractMatFormField<LookupItem>
+  implements OnInit, OnDestroy
+{
+  public control = new FormControl();
+  filteredLookupItemsObservable: Observable<LookupItem[]> = of([]);
```

Comment:
`public`?

Reply:
Neds to be public as its being used in html